db/migrate/20201021065138_add_prefill_source_to_person_escort_records.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrefillSourceToPersonEscortRecords < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :person_escort_records, :prefill_source, type: :uuid, index: true, null: true
```

Comment:
Do we need the index?

Reply:
yea, I think its good practice to index references, even self references